installer/charts/rhtap-dh/templates/plugins-content.yaml

```diff
@@ -146,4 +146,20 @@ plugins:
     package: ./dynamic-plugins/dist/backstage-plugin-techdocs
   - disabled: false
     package: ./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic
+
+  #
+  # RBAC
+  #
+  {{- $githubSecretObj := (lookup "v1" "Secret" $integrationNamespace "rhtap-github-integration") }}
```

Comment:
You don't have to retrieve the secret, you can just use the `$GITHUB__URL` set in `extra-env.yaml`.

Reply:
hum... plugin config will be able to use those env var? I was not sure cause there is no reference to env vars at the moment. Let me try it.

Reply:
Done, works well, thanks